frontend/src/metabase/visualizer/visualizer.slice.ts

```diff
@@ -207,12 +203,8 @@ const fetchCardQuery = createAsyncThunk<Dataset, CardId>(
   },
 );
 
-export const setDisplay = createAsyncThunk(
+export const setDisplay = createAction<VisualizationDisplay>(
   "visualizer/setDisplay",
-  (display: VisualizationDisplay | null, { getState }) => {
-    const computedSettings = getVisualizerComputedSettings(getState());
-    return { display, computedSettings };
-  },
 );
```

Comment:
Since it's a plain action creator now, I think it'd be more consistent if we put it on `visualizerHistoryItemSlice` similarly to `setTitle`, `updateSettings`, etc.